packages/application-config/src/load-config.ts

```diff
@@ -37,52 +37,97 @@ const loadJsModule: LoaderSync = (filePath) => {
   return JSON.parse(output);
 };
 
-const moduleName = 'custom-application-config';
-const explorer = cosmiconfigSync(moduleName, {
-  // Restrict the supported file formats / names
-  searchPlaces: [
-    `.${moduleName}rc`,
-    `.${moduleName}.json`,
-    `.${moduleName}.js`,
-    `.${moduleName}.cjs`,
-    `.${moduleName}.mjs`,
-    `.${moduleName}.ts`,
-    `${moduleName}.json`,
-    `${moduleName}.js`,
-    `${moduleName}.cjs`,
-    `${moduleName}.mjs`,
-    `${moduleName}.ts`,
-  ],
-  loaders: {
-    noExt: defaultLoaders['.json'],
-    '.js': loadJsModule,
-    '.cjs': loadJsModule,
-    '.mjs': loadJsModule,
-    '.ts': loadJsModule,
-  },
-});
+const getExplorer = (configFileName: string) => {
+  return cosmiconfigSync(configFileName, {
+    // Restrict the supported file formats / names
+    searchPlaces: [
+      `.${configFileName}rc`,
+      `.${configFileName}.json`,
+      `.${configFileName}.js`,
+      `.${configFileName}.cjs`,
+      `.${configFileName}.mjs`,
+      `.${configFileName}.ts`,
+      `${configFileName}.json`,
+      `${configFileName}.js`,
+      `${configFileName}.cjs`,
+      `${configFileName}.mjs`,
+      `${configFileName}.ts`,
+    ],
+    loaders: {
+      noExt: defaultLoaders['.json'],
+      '.js': loadJsModule,
+      '.cjs': loadJsModule,
+      '.mjs': loadJsModule,
+      '.ts': loadJsModule,
+    },
+  });
+};
+
+const customApplicationExplorer = getExplorer('custom-application-config');
+const customViewExplorer = getExplorer('custom-view-config');
```

Comment:
We need to be able to load config files for both Custom Applications and Custom Views